biz.aQute.bnd.reporter/bnd.bnd

```diff
@@ -30,29 +43,16 @@ Export-Package: \
 	biz.aQute.bnd.reporter.generator,\
 	biz.aQute.bnd.reporter.manifest.dto
 
-Import-Package: \
-    sun.*; resolution:=optional,\
-    com.google.appengine.*; resolution:=optional,\
-    com.google.apphosting.*; resolution:=optional,\
-    *
-
 -conditionalpackage: \
 	aQute.lib.*,\
-	aQute.libg.*,\
-	com.google.common.*,\
-	com.googlecode.*,\
-	org.apache.commons.lang3.*	
+	aQute.libg.*	
 	
 Private-Package: \
-	org.parboiled.*;-split-package:=merge-first,\
-	org.objectweb.asm.*,\
-	org.jtwig.*,\
 	biz.aQute.bnd.reporter.plugins.*,\
 	biz.aQute.bnd.reporter.generator,\
 	biz.aQute.bnd.reporter.helpers,\
-	biz.aQute.bnd.reporter.component.dto,\
-	biz.aQute.bnd.reporter.manifest.dto
```

Comment:
Perhaps this whole thing should be `-includepackage: biz.aQute.bnd.reporter.*`?

Reply:
Yes, I will replace it.